src/components/screens/main-screen.tsx

```diff
@@ -231,11 +323,46 @@ const MainScreen: React.FC = () => {
   /**
    * Renders courses
    */
-  const renderCourses = () => (
-    <Stack spacing={3}>
-      {courses.map(course => <CourseCard course={course}/>)}
-    </Stack>
-  );
```

Comment:
Why is this removed?

Reply:
Nono this chunck of code is present. I see your comment on the old version